### DISCUSSION TITLE
WARNING: cannot connect to daemon at ws://localhost:55400

### DISCUSSION THREAD

**Discussion by tksalarzai** [Chia-Network/chia-blockchain#4697] [Support]
My harvester logs showing these error again and again "**WARNING: cannot connect to daemon at ws://localhost:55400**" and still the harvestor plots are showing in full node gui 0/4 plots and also in gui connected harvester nodes, but dont know what this error is ? please help
![Screenshot 2021-05-12 193528](https://user-images.githubusercontent.com/77279839/117995000-6651d680-b35a-11eb-86ba-fa2d553f35a5.png)

![Screenshot 2021-05-12 194537](https://user-images.githubusercontent.com/77279839/117995478-cb0d3100-b35a-11eb-9f8e-023c05bfa07e.png)

![Screenshot 2021-05-12 194451](https://user-images.githubusercontent.com/77279839/117995330-ab760880-b35a-11eb-8e45-1ab7355a875b.png)

**Comment by GeorgePapageorgakis** [comment]
Still no updates/response from support on the matter...

**Comment by BrandtH22** [comment]
@tksalarzai , if you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
